app/commands/webhooks/paypal/request_access_token.rb

```diff
@@ -0,0 +1,41 @@
+require 'uri'
+require 'net/http'
+
+class Webhooks::Paypal::RequestAccessToken
+  class RequestAccessTokenError < RuntimeError; end
+
+  include Mandate
+
+  def call
+    response = http.request(request)
+
+    raise RequestAccessTokenError unless response.is_a?(Net::HTTPSuccess)
+
+    response_body = JSON.parse(response.body, symbolize_names: true)
+    response_body[:access_token]
+  rescue StandardError
+    raise RequestAccessTokenError
+  end
+
+  private
+  memoize
+  def http
+    http = Net::HTTP.new(OAUTH2_TOKEN_URI.host, OAUTH2_TOKEN_URI.port)
+    http.use_ssl = true
+    http
+  end
+
+  def request
+    request = Net::HTTP::Post.new(OAUTH2_TOKEN_URI.request_uri)
+    request.basic_auth(paypal_client_id, paypal_client_secret)
+    request.set_content_type('application/x-www-form-urlencoded')
+    request.set_form_data({ 'grant_type' => 'client_credentials' })
+    request
+  end
+
+  def paypal_client_id = ENV.fetch('PAYPAL_CLIENT_ID', Exercism.secrets.paypal_client_id)
+  def paypal_client_secret = ENV.fetch('PAYPAL_CLIENT_SECRET', Exercism.secrets.paypal_client_secret)
```

Comment:
Can we get rid of ENV here and use the new locals thing instead pls?

Reply:
Ah nice: https://github.com/exercism/config/commit/4a8e0100c9dbeb294900adb91420c9910593e08a